### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by kiboma2021**
- Create a database named vet_clinic.
- Create a table named animals with the following columns:

1. id: integer
2. name: string
3. date_of_birth: date
4. escape_attempts: integer
5. neutered: boolean
6. weight_kg: decimal

- Copy the SQL you used in the previous point into the schema.sql file in the generated repository.
- Insert the following data:

1. Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
2. Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
3. Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
4. Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.

- Copy the SQL you used in the previous point into the data.sql file in the generated repository.
- Write queries for the following quests:

1. Find all animals whose name ends in "mon".
2. List the name of all animals born between 2016 and 2019.
3. List the name of all animals that are neutered and have less than 3 escape attempts.
4. List the date of birth of all animals named either "Agumon" or "Pikachu".
5. List name and escape attempts of animals that weigh more than 10.5kg
6. Find all animals that are neutered.
7. Find all animals not named Gabumon.
8. Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

- Copy the SQL you used in the previous point into the queries.sql file in the generated repository.

**Screenshots**
SQL queries1

SQL queries2

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```
Comment (ShahierNashaat) [inline queries.sql]: - [ ] Kindly be noticed that the question is `find all animals whose name ends in "mon"?` so you should write the query like this.
```sql
`SELECT * FROM animals WHERE name LIKE '%mon';`
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @kiboma2021, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (kiboma2021) [reply queries.sql]: @ShahierNashaat , Thank you for the correction.

**Review by addod19 — APPROVED** [review]
### Hi @kiboma2021, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the query :100: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**